src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -168,6 +170,36 @@ internal void LoadContext(ExecutionContext context, bool checkReturnValue)
             LoadContext(context);
         }
 
+        public ExecutionContext LoadContract(ContractState contract, string method, CallFlags callFlags)
+        {
+            ContractMethodDescriptor md = contract.Manifest.Abi.GetMethod(method);
+            if (md is null) return null;
+
+            ExecutionContext context;
+            if (NativeContract.IsNative(contract.ScriptHash))
+            {
+                using ScriptBuilder sb = new ScriptBuilder();
+                sb.Emit(OpCode.DEPTH, OpCode.PACK);
+                sb.EmitPush(md.Name);
+                context = LoadScript(sb.ToArray(), CallFlags.None);
+            }
+            else
+            {
+                context = LoadScript(contract.Script, callFlags, md.Offset);
+            }
+
+            // Call initialization
+
+            var init = contract.Manifest.Abi.GetMethod("_initialize");
+
+            if (init != null)
+            {
+                LoadContext(context.Clone(init.Offset), false);
+            }
```

Comment:
These code can be moved to the `else` block of `IsNative`. Because native contracts don't have `_initialize` method.

Reply:
Done